include/effects/matrix/PatternSMSupernova.h

```diff
@@ -24,10 +24,8 @@ class PatternSMSupernova : public LEDStripEffect
 
     void Start() override
     {
+        std::for_each(_debris_items.begin(), _debris_items.end(), [](DebrisItem& debris_item) { debris_item.Clear(); });
         g()->Clear();
-        enlargedDebrisItemNUM = 200;
-        if (enlargedDebrisItemNUM > MAX_COUNT)
-            enlargedDebrisItemNUM = 200U;
```

Comment:
Is this the same 200 as the debris count define below?  I don't think they're liinked?

Reply:
I didn't change the debris count magic to something I came up with myself. I reverted to the value it had before #489. At that time there was an index "if" that ran up to the value of `enlargedDebrisItemNUM`, thus 200.